crates/ignore/src/default_types.rs

```diff
@@ -167,7 +167,10 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("pascal", &["*.pas", "*.dpr", "*.lpr", "*.pp", "*.inc"]),
     ("pdf", &["*.pdf"]),
     ("perl", &["*.perl", "*.pl", "*.PL", "*.plh", "*.plx", "*.pm", "*.t"]),
-    ("php", &["*.php", "*.php3", "*.php4", "*.php5", "*.phtml"]),
+    ("php", &[
+        "*.php", "*.php3", "*.php4", "*.php5", "*.php7", "*.php8",
```

Comment:
What about `*.php6`?

Reply:
PHP 6 doesn't exist [1], but we could include the extension as well. :) 

[1] https://wiki.php.net/rfc/php6

Reply:
Haha. Nice. OK, yes, leave it out. Could you add a small comment with a link to that page in the source here? I'm sure I won't be the only one to notice and ponder about why `.php6` is absent.